client/src/map.js

```diff
@@ -245,10 +263,12 @@ function Map(props) {
               props.wsSend(JSON.stringify(routeRequest("30")));
             }}
             icon={{
-              url: "/bus.svg",
-              origin: new window.google.maps.Point(0, 0),
-              anchor: new window.google.maps.Point(15, 15),
-              scaledSize: new window.google.maps.Size(30, 30),
+              path: "M25.5,8.25H23.22V3H4.82V8.25H2.5V9.53H4.82V51.34A1.67,1.67,0,0,0,6.48,53h15.1a1.65,1.65,0,0,0,1.64-1.65V9.53H25.5Z",
```

Comment:
Vad är dehär? Har de något med den nya ikonen att göra?

Reply:
Det är pathen från small_simple_bus.svg. Anledningen till att inte filen länkas in är att man inte kan sätta rotation attributet för ikonen om man gör så.

Reply:
Alright 👍